src/js/components/covid19/OpportunityProjectPage.jsx

```diff
@@ -0,0 +1,119 @@
+                            <p><span className="bold">THE OPPORTUNITY</span> – The correlation between federal spending data and demographic data is critical to understanding how federal funds are distributed to various communities, but this correlation has not been fully explored or made accessible to local stakeholders. This creates blind spots for government leaders as they make decisions on funding allocation and addressing inequities in their communities. To help address this need, the opportunity project sought technologists to help innovate and build digital products with experts (known as user advocates) on the ground in communities across America.</p>
+                            <p><span className="bold">THE TARGET END USERS</span> – Local government officials, federal funding decision makers, equity policy experts, community advocacy organizations and data journalists.</p>
+                            <p><span className="bold">THE VISION</span> – By bringing together Census, Treasury, and other open data sources, there can be innovative digital tools built to help stakeholders better understand whether the federal government is equitably distributing federal funds during the COVID-19 pandemic.</p>
+                            <p><span className="bold">THE RESULTS</span> – The Opportunity Project 2021 sprint concluded in November 2021.  Working with user advocates, data stewards, product advisors and following Census’s TOP product development toolkit, tech teams submitted innovative tools to analyze equity in federal COVID-19 spending.  The results are both inspiring and illuminating.  To see what tech teams built, please visit the website links below.</p>
+                            <p><span className="bold">PLEASE NOTE: Inclusion on this page does not constitute endorsement by the Department of the Treasury, Bureau of the Fiscal Service, or any employees thereof of any product, service, or enterprise.</span></p>
+
+                            <p className="bold underline">Bowie State University</p>
+                            <p>Product statement: Bowie State University Opportunity Project uses publicly accessible CDC’s Social Vulnerability Index, CDC’s County vaccination rates, the ARP county COVID-19 vaccine spending data, and Co-est 2020 to assess COVID-19 vaccination and equity problems for community leaders as end users.</p>
+                            <p>Link: {externalLink("https://app.flow.gl/flow/3xfvr3", "Bowie State University TOP Project on Flow Immersive")}</p>
+
+                            <p className="bold underline">Morehouse College</p>
```

Comment:
Should these use an h tag instead of `<p>`?

Reply:
H tags should never be used since CSS was created. They cause confusion and headaches for screen readers if they aren't properly nested.

Reply:
I would argue semantic HTML is _better_ for screen readers to correctly identify page layout. This is not a paragraph, it's a heading. They should be properly nested and can be styled however we like.

Reply:
https://www.deque.com/blog/javascript-frameworks-the-lost-art-of-html/

Reply:
Valid points. I'll update

Reply:
Thank you!